packages/profile-picture/src/profile-picture.ts

```diff
@@ -49,6 +49,10 @@ export class ProfilePictureElement extends LitElement {
   }
 
   updated(changedProps) {
+    if (changedProps.has('personId')) {
+      this.refresh();
```

Comment:
@SketchBookkeeper I'm not sure we should use refresh() here. We don't actually need to increment the cache-buster. Just set hasError to false.

Reply:
Agreed, I bet there is a better way to fix this. maybe just `hasErrors = false`